recipe/meta.yaml

```diff
@@ -12,17 +12,17 @@ source:
   md5: {{ md5 }}
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
 
 requirements:
   build:
     - cmake
     - pkg-config  # [unix]
-    - qt 
+    - qt 4.8.*
   run:
-    - qt
- 
+    - qt 4.8.*
```

Comment:
I think the difference is for Qt/PyQt. Qt is `4.8.*`, but PyQt is `4.11.*`. No clue why that differences makes sense, but those are the latest releases in the `4.*` series.